src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) throws CloneNotSupportedException {
+

```

Comment:
you don't need this black line as well as line 25

Reply:
Fixed.